airbyte-integrations/connectors/destination-redshift/src/test-integration/java/io/airbyte/integrations/destination/redshift/RedshiftStagingS3DestinationAcceptanceTest.java

```diff
@@ -120,7 +127,7 @@ public void testCheckIncorrectDataBaseFailure() throws Exception {
   public void testGetFileBufferDefault() {
     final RedshiftStagingS3Destination destination = new RedshiftStagingS3Destination();
     assertEquals(destination.getNumberOfFileBuffers(config),
-        FileBuffer.DEFAULT_MAX_CONCURRENT_STREAM_IN_BUFFER);
+            FileBuffer.DEFAULT_MAX_CONCURRENT_STREAM_IN_BUFFER);
```

Comment:
nit: I think this can be one line, not sure why the this wasn't previously the case since L137, L147 are the same.

Reply:
Done